homework/shelkovich_ilya/functional_interfaces/src/main/java/com/tutrit/java/quickstart/Functions.java

```diff
@@ -0,0 +1,29 @@
+package com.tutrit.java.quickstart;
+
+import com.tutrit.java.quickstart.bean.Person;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.LocalDateTime;
+import java.util.function.*;
+
+public class Functions {
+
+    private Functions() {
+
+    }
+
+    static Logger log = LoggerFactory.getLogger(Functions.class);
+
+    public static final Runnable printCurrentTime = () -> log.info("{}", LocalDateTime.now());
```

Comment:
According Java Code Convencion if fields has group of modifiers  "public static final" then field are writing as UPPER CASE

Reply:
woah, didn't know that, thx